composer.json

```diff
@@ -4,7 +4,7 @@
     "license": "MIT",
     "require": {
         "php": "7.1.*",
-        "nikic/php-parser": "^3.0.5",
+        "nikic/php-parser": "^3.1.0",
         "phpdocumentor/reflection-docblock": "^3.2.0",
         "phpdocumentor/type-resolver": "^0.4.0",
```

Comment:
Are these deps still needed then?

Reply:
@Ocramius Yes, there are needed for https://github.com/Roave/BetterReflection/blob/master/src/Reflection/ReflectionFunctionAbstract.php#L427 etc.